cl_sii/rut/crypto_utils.py

```diff
@@ -36,10 +36,10 @@ def get_subject_rut_from_certificate_pfx(pfx_file_bytes: bytes, password: Option
         results = [
             x.value
             for x in subject_alt_name_ext.value._general_names
-            if x.type_id == constants.SII_CERT_TITULAR_RUT_OID
+            if hasattr(x, 'type_id') and x.type_id == constants.SII_CERT_TITULAR_RUT_OID
         ]
-    except AttributeError:
-        raise Exception('Certificate has no RUT information')
+    except AttributeError as exc:
```

Comment:
@fpinto-cdd I think with the check `if hasattr(x, 'type_id')`, this exception would no longer be raised, right?

Reply:
It will not be raised because of this, but could be trigger for another kind of malformed certificate (for example, one without the `GeneralNames` object). It should not occur anyway.

Reply:
You're right... 👌